cursorless-snippets/functionDeclaration.cursorless-snippets

```diff
@@ -0,0 +1,49 @@
+{
+  "functionDeclaration": {
+    "definitions": [
+      {
+        "scope": {
+          "langIds": [
+            "typescript",
+            "typescriptreact",
+            "javascript",
+            "javascriptreact"
+          ]
+        },
+        "body": [
+          "function $name($parameterList) {",
+          "\t$body",
+          "}"
+        ],
+        "variables": {
+          "name": {
+            "formatter": "camelCase"
```

Comment:
After discussion with @AndreasArvidsson, we're inclined to use Talon formatters instead.  There are two ways we thought of to do this:

- Could use a knausj comma-separated formatter string in this field, and then Talon would query Cursorless to figure out which formatter to use and do the formatting itself
- Could drop this field here, instead adding a column to the Talon csv that indicates which type of formatter to use.  It will support things like `user.code_public_function_formatter` so that it can be language-dependant.  The problem with this approach is that it will be hard to support cross-split insertion once we support that